src/meta/src/hummock/manager/mod.rs

```diff
@@ -309,6 +308,11 @@ where
                 object_store.clone(),
             )
             .await?;
+
+            // config bucket lifecycle for new cluster.
+            if let risingwave_object_store::object::ObjectStoreImpl::S3(s3) = object_store.as_ref() && !env.opts.do_not_config_object_storage_lifecycle{
+            s3.inner().configure_bucket_lifecycle().await;
```

Comment:
How does this if block pass `cargo fmt` check? 🤔 
The indentation seems wrong

Reply:
wired, let me see.

Reply:
`cargo fmt` pass, let me modify it manually.